pkg/k8s/kubernetes.go

```diff
@@ -637,6 +733,27 @@ func GetPods(c *kubernetes.Clientset, dp DeploymentParams) (corev1.PodList, erro
 	return npl, nil
 }
 
+func GetNakedPods(c *kubernetes.Clientset, label string) (corev1.PodList, error) {
+	npl := corev1.PodList{}
+	listOpt := metav1.ListOptions{
+		LabelSelector: label,
+	}
```

Comment:
If you use a fieldSelector you don't need to check if the pods are healthy

```suggestion
	listOpt := metav1.ListOptions{
		LabelSelector: label,
		FieldSelector "status.phase=Running":
	}
```